src/doc/src/reference/overriding-dependencies.md

```diff
@@ -291,6 +291,10 @@ also be patched with versions of crates that already exist. If a source is
 patched with a crate version that already exists in the source, then the
 source's original crate is replaced.
 
+Cargo only looks at the patch settings in the `Cargo.toml` manifest at the
+root of the workspace. Patch settings defined in dependencies will be
+ignored.
```

Comment:
I fell like something stating that when cargo is run in a workspace this is how it works

```suggestion
When Cargo is run within a workspace it only looks at the patch settings in the `Cargo.toml` manifest at the
root of the workspace. Patch settings defined in dependencies will be
ignored.
```

Reply:
I was maintaining the existing way of framing it as contained in the `profile` documentation.  Is it important enough for us to re-review that documentation and address it in this same PR or can someone take this on outside of this PR?

Reply:
I think an outside PR would be fine